extensions/quartz/runtime/src/main/java/io/quarkus/quartz/runtime/QuartzScheduler.java

```diff
@@ -56,8 +59,15 @@
     private final AtomicInteger triggerNameSequence;
     private final Map<String, ScheduledInvoker> invokers;
 
+    @Produces
+    @Singleton
```

Comment:
Should we use `@DefaultBean` here?

Reply:
I don't think so. Why?

Reply:
When the user code `@Produces` their own `org.quartz,Scheduler`/?

Reply:
But maybe we shouldn't allow that anyway :)